changes/566.feature.md

```diff
@@ -0,0 +1 @@
+Using python-dotenv for auto detect .env file
```

Comment:
Please use the imperative form of sentence and describe not just what you did but just what it changes in the perspective of users.